config/environments/development.rb

```diff
@@ -12,7 +12,7 @@
   # In the development environment your application's code is reloaded on
   # every request. This slows down response time but is perfect for development
   # since you don't have to restart the web server when you make code changes.
-  config.cache_classes = ENV.fetch('CACHE_CLASSES', 'true') == 'true'
+  config.cache_classes = ENV.fetch('CACHE_CLASSES', 'false') == 'true'
```

Comment:
I think the defaults are supposed to be safe to deploy to production so that we're not relying on the config in production to ensure the behaviour we want.  So a better solution to this would be to change the default in the environment dot file so that it sets `CACHE_CLASSES` to `false` for development.

Reply:
Oh sorry, I've just noticed this file is development specific

Reply:
Weird, because the comment above the line kind of hints that it does the opposite of what is happening.

Reply:
Rather than comparing to a specific string `true`, I wonder if there's any way to check that the environment variable was set at all.  Not having any value set on it should be like saying `false` while having any value set should be like `true`.  IMO anyway

Reply:
Ahh well that explains the comment!  Must have been copy pasted without updating when the logic was inverted.  I guess check with the team why we would be doing caching by default on Sequencescape.  Perhaps it can cause issues.  Don't want to break everyone's development environment and then they need to set an environment variable.